src/iOS/Xamarin.iOS/Samples/GraphicsOverlay/SurfacePlacements/SurfacePlacements.cs

```diff
@@ -154,6 +191,14 @@ public override void LoadView()
             _picker.SelectedSegment = 0;
             _picker.TranslatesAutoresizingMaskIntoConstraints = false;
 
+            _zSlider = new UISlider() { TranslatesAutoresizingMaskIntoConstraints = false };
+            _zSlider.MinValue = 0;
+            _zSlider.MaxValue = 140;
+            _zSlider.SetValue(70, false);
+
+            _valueLabel = new UILabel() { Text = "70 meters", TranslatesAutoresizingMaskIntoConstraints = false };
+
+
```

Comment:
I'd remove the extra blank line.

Also, this is a nitpick, but when using an object initializer, parenthesis can be omitted, e.g. `_zSlider = new UISlider { TranslatesAutoresizingMaskIntoConstraints = false };`